pvlib/bifacial/loss_models.py

```diff
@@ -101,6 +101,14 @@ def power_mismatch_deline(
        The global irradiance RMAD is different from the backside irradiance
        RMAD.
 
+    RMAD of a variable :math:`G_{total}` is defined as:
```

Comment:
About 20 lines, up, change "Relative Mean Absolute Difference [unitless]" to "Relative Mean Absolute Difference (RMAD) [unitless]"